README.md

```diff
@@ -23,350 +23,46 @@
 <img src="assets/github-hr.png" alt="---" />
 <br/>
 
-**k6** is a modern load testing tool, building on [our](https://k6.io/about) years of experience in the load and performance testing industry. It provides a clean, approachable scripting API, [local](https://k6.io/docs/getting-started/running-k6) and [cloud execution](https://k6.io/docs/cloud), and flexible configuration.
+**k6** is a modern load-testing tool, building on [our years of experience](https://k6.io/about) in the performance and testing industries.
+k6 is built to be simple, powerful, and extensible. Every commit to this repo aims to help create an application with **high performance and excellent developer experience**.
+Some key features include:
 
-This is how load testing should look in the 21st century.
+- **Configurable load generation.** Even lower-end machines can simulate lots of traffic.
+- **Tests as code.** Reuse scripts, modularize logic, version control, and integrate tests with your CI.
+- **A full-featured API.** The scripting API is packed with features that help you simulate real application traffic.
+- **An embedded JavaScript engine.** The performance of Go, the scripting simplicity of vanilla ECMAscript (without baggage from the wider JS ecosystem).
+- **Multiple Protocol support**. HTTP, WebSockets, gRPC, and more.
+- **Large extension ecosystem.** You can extend k6 to support your needs. And many people have already shared their extensions with the community!
+- **Flexible metrics storage and visualization**. Summary statistics or granular metrics, exported to the service of your choice.
+
+This is what load testing looks like in the 21st century.
 
 <p align="center">
   <img width="600" src="assets/k6-demo.gif">
```

Comment:
This Demo is 3 years old and likely should be updated.

Also as a recommendation can we not store "assets" that aren't part of the project in the repo and instead upload them to github by just copying them in an issue and then using the generated link (no need to post the issue).